guides/common/modules/proc_configuring-group-mapping-for-keycloak-authentication.adoc

```diff
@@ -0,0 +1,19 @@
+[id="configuring-group-mapping-for-keycloak-authentication_{context}"]
+= Configuring Group Mapping for {Keycloak} Authentication
+
+Optionally, to implement the Role Based Access Control (RBAC), create a group in {Project}, assign a role to this group, and then map an Active Directory group to the {Project} group.
+As a result, anyone in the given group in {Keycloak} are logged in under the corresponding {Project} group.
+This example configures users of the {Project}-admin user group in the Active Directory to authenticate as users with administer privileges on {Project}.
+
+.Procedure
+
+. In the {Project} web UI, navigate to *Administer* > *User Groups*, and click the *Create User Group* button.
+. In the *Name* field, enter a name for the user group.
+The name should not be the same as in the Active Directory.
+. Do not add users and user groups to the right-hand columns.
+Click the *Roles* tab.
+. Select the *Administer* check box.
+. Click the *External Groups* tab.
+. Click the *Add external user group* button.
```

Comment:
. Click *Add external user group*.
(IBM style guide, for buttons)

Reply:
What do you mean by this? This is a tab, not a button in the UI.

Reply:
Ok, then change it to tab, similar to the previous step- 
. Click the *Add external user group* tab.?

Reply:
Oh sorry, I was looking at a wrong line. Will fix this now.